example-config.toml

```diff
@@ -42,6 +42,7 @@ Project = "<your project>"
 Dataset = "<your dataset>"
 ExportTable = "fs_export"
 SyncTable = "fs_sync"
+PartitionExpiration = "0"
```

Comment:
Just making sure this is backward-compatible with older hauser configs: when this line is omitted, the config struct gets default values, so the expiration duration is 0. And zero means don't expire, correct?

Also, in order for this to parse correctly, this is the usual go shorthand for durations, correct? Might be nice to put a comment here to explain or link to that syntax, since so many hauser users are not overly familiar with go.

Reply:
> Just making sure this is backward-compatible with older hauser configs: when this line is omitted, the config struct gets default values, so the expiration duration is 0. And zero means don't expire, correct?

This is correct. I tested with a real table to make sure.

> Also, in order for this to parse correctly, this is the usual go shorthand for durations, correct? Might be nice to put a comment here to explain or link to that syntax, since so many hauser users are not overly familiar with go.

Good idea, will update!